Makefile.dep

```diff
@@ -606,6 +606,14 @@ ifneq (,$(filter random,$(USEMODULE)))
   endif
 endif
 
+ifneq (,$(filter random_seed,$(USEMODULE)))
+  USEMODULE += luid
+  USEMODULE += xtimer
+
+  FEATURES_REQUIRED += periph_hwrng
+  FEATURES_REQUIRED += periph_adc
```

Comment:
Hmm, these dependencies still don't reflect whats actually done in the module. What should be reflected here is the actual usage and dependency of the modules used: 
`if (hwrng available) then (hwrng) else luid AND if (adc available) then adc else xtimer` (sorry, hard to read...).